fdm-app/vite.config.ts

```
@@ -3,17 +3,22 @@ import { sentryVitePlugin } from "@sentry/vite-plugin"
 import { defineConfig } from "vite"
 import tsconfigPaths from "vite-tsconfig-paths"
 
+// Vite does not support loading config files from `/lib/config`...
+const org = process.env.VITE_SENTRY_ORG
+const authToken = process.env.SENTRY_AUTH_TOKEN
+const project = process.env.VITE_SENTRY_PROJECT
+let pluginSentry: any
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)